app/controllers/v0/users_controller.rb

```diff
@@ -19,8 +19,9 @@ def icn
 
     def credential_emails
       emails = current_user.user_account.user_verifications.each_with_object({}) do |verification, credentials|
-        credentials[verification.credential_type.to_sym] =
-          verification.user_credential_email.credential_email
+        next unless verification.user_credential_email
```

Comment:
Kind of a nitpick but I think it would be a bit cleaner if we select the `user_verifications` we want first instead of looping through them
```ruby
verifications = current_user.user_account.user_verifications.select { |v| v.user_credential_email }

emails = verifications.to_h { |v|  [v.credential_type, v.user_credential_email.credential_email] }
```

Reply:
or for that matter I suppose we could do `user_verifications.where.not(user_credential_email: nil)`, and just do it with the query